test/test_jit_fuser.py

```diff
@@ -616,6 +618,9 @@ def iou(b1x1, b1y1, b1x2, b1y2, b2x1, b2y1, b2x2, b2y2):
     @unittest.skipIf(not RUN_CUDA_MULTI_GPU, "needs non-zero device")
     @enable_cpu_fuser
     def test_fusion_reuse_multi_gpu(self):
+        if graph_executor_mode() == ProfilingMode.LEGACY:
```

Comment:
most tests should work with the legacy executor, i'm not sure i understand why there are failures now.